### PR TITLE
Added Graph Verification Support in ingestion report

### PR DESCRIPTION
**Pull request by animuk**

### CODE REVIEW

File: f8a_report/report_helper.py [hunk 1]
```diff
@@ -540,66 +538,75 @@ def normalize_ingestion_data(self, start_date, end_date, ingestion_data, frequen
...
+            all_epv_list_v2[epv_arr[0]][epv_arr[1]]['package_known'] = values
+            all_epv_list_v2[epv_arr[0]][epv_arr[1]]['versions'] = []
+            versn_template['version'] = epv_arr[2]
+            versn_template['ingested_in_graph'] = values
+            if values == 'false':
+                failed_deps_count['all'] = failed_deps_count['all'] + 1
+                failed_deps_count[epv_arr[0]] = failed_deps_count[epv_arr[0]] + 1
+            all_epv_list_v2[epv_arr[0]][epv_arr[1]]['versions'].append(versn_template)
+
+        logger.error(graph_input)
```
Comment (yzainee) [inline f8a_report/report_helper.py]: what does this error message denote here?

File: f8a_report/report_helper.py [hunk 2]
```diff
@@ -540,66 +538,75 @@ def normalize_ingestion_data(self, start_date, end_date, ingestion_data, frequen
...
-            if data['ecosystem'] == 'maven':
-                data['failed_epvs'] = failed_epv_list['maven']
-            elif data['ecosystem'] == 'npm':
-                data['failed_epvs'] = failed_epv_list['npm']
-            elif data['ecosystem'] == 'python':
-                data['failed_epvs'] = failed_epv_list['python']
+            graph_input.append(graph_template)
+
+        graph_output = generate_report_for_latest_version(graph_input)
+        logger.error(graph_output)
```
Comment (yzainee) [inline f8a_report/report_helper.py]: logger.error?

File: f8a_report/report_helper.py [hunk 3]
```diff
@@ -540,66 +538,75 @@ def normalize_ingestion_data(self, start_date, end_date, ingestion_data, frequen
...
             'ingestion_summary': {},
-            'ingestion_details': []
+            'ingestion_details_v2': {}
         }
 
         all_deps_count = {'all': 0, 'npm': 0, 'maven': 0, 'python': 0}
         failed_deps_count = {'all': 0, 'npm': 0, 'maven': 0, 'python': 0}
-        all_epv_list = {'npm': [], 'maven': [], 'python': []}
-        failed_epv_list = {'npm': [], 'maven': [], 'python': []}
+        all_epv_list_v2 = {'npm': {}, 'maven': {}, 'python': {}}
```
[the remark this answers is not in the document]
Comment (yzainee) [reply f8a_report/report_helper.py]: same comment is applicable for the count logic as well

File: f8a_report/report_helper.py (hunk 2 above)
Comment (animuk) [reply f8a_report/report_helper.py]: That was for debugging. Removing it.

File: f8a_report/report_helper.py (hunk 1 above)
Comment (animuk) [reply f8a_report/report_helper.py]: That was for debugging. Removing it.

File: f8a_report/report_helper.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (animuk) [reply f8a_report/report_helper.py]: made it all dynamic

**Comment by miteshvp** [comment]
@animuk - it is a good practice to not merge your own PRs unless you get an approval from the reviewers.

**Comment by animuk** [comment]
@miteshvp I thought, yusuf is already reviewed the PR, So this PR can be merged. Fine. If you have any additional review comments, please let me know.I will take care of it.

**Comment by miteshvp** [comment]
@yzainee has to approve the request which is not the case.

**Pull request merged**